src/main/java/com/splunk/logging/HttpEventCollectorLog4jAppender.java

```diff
@@ -73,12 +74,12 @@ private HttpEventCollectorLog4jAppender(final String name,
                          final String eventBodySerializer)
     {
         super(name, filter, layout, ignoreExceptions);
-        Dictionary<String, String> metadata = new Hashtable<String, String>();
```

Comment:
Thank you for cleaning this up! I think this is carryover from the .net LL since `Dictionary` is the commonly used map type